Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "EDF"
 uuid = "ccffbfc1-f56e-50fb-a33b-53d1781b2825"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.5.1"
+version = "0.6.0"
```

Comment:
I'd argue that this is a bugfix, but I guess it could potentially be breaking in edge cases?

Reply:
Yeah, the previous behavior let you actually construct/manipulate TimestampedAnnotationLists with "unrepresentable" values even if you couldn't write them, so technically is breaking; figure better safe than sorry